### PR TITLE
Video settings indicator animation

### LINKED ISSUE #2385: Video Animation ⚠️: Change Playback Settings
## Context

Animation for submitting settings choices

## Scope
- [ ] Animation 1:58 from https://www.loom.com/share/cd7f7a5123e74f1fbf3f096fd32fb5e2

## Designs

Animation:
https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2090%3A120610

## Additional Info

Broader usecase:
https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2127%3A135540
Design of full feature:
https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2090%3A120527

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #2385 

**Design**
https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2127%3A135540

### CODE REVIEW

File: packages/atlas/src/components/_video/VideoPlayer/ControlsIndicator.tsx [hunk 1]
```diff
@@ -51,10 +54,24 @@ export const ControlsIndicator: FC<ControlsIndicatorProps> = ({ player, isLoadin
...
       // if we omit this we'll get stale results
       timeout = window.setTimeout(() => {
-        const indicator = createIndicator(e.type as VideoEvent, player.volume(), player.muted())
+        const indicator = createIndicator(
+          e.type as VideoEvent,
+          player.volume(),
+          player.muted(),
+          selectedLanguage,
+          player.playbackRate()
+        )
```
Comment (drillprop) [inline packages/atlas/src/components/_video/VideoPlayer/ControlsIndicator.tsx]: I'm thinking, maybe we could just pass `player` here? Function with 5 parameters feels awkward

File: packages/atlas/src/components/_video/VideoPlayer/VideoPlayer.styles.ts [hunk 2]
```diff
@@ -544,6 +573,12 @@ export const Container = styled.div<ContainerProps>`
         }
       }
 
+      ${ControlsIndicatorWrapper} {
+        transform: translateY(
+          ${({ isSettingsPopoverOpened }) => (isSettingsPopoverOpened ? HOVERED_CONTROLS_INDICATOR_POSITION : 0)}
+        );
+      }
+
```
Comment (drillprop) [inline packages/atlas/src/components/_video/VideoPlayer/VideoPlayer.styles.ts]: As I said in our meeting - let's consult this with a designer first. Maybe there is a better solution

**Review by drillprop — CHANGES_REQUESTED** [review]
The indicator doesn't show when you press `C` on the keyboard and I think it should.

**Review by drillprop — CHANGES_REQUESTED** [review]
Unfortunately, it's still a little bit buggy :/ 

https://user-images.githubusercontent.com/51168865/186358085-5567a07e-ce62-4434-9327-38fe726c0e14.mov

**Review by drillprop — APPROVED** [review]
LGTM

**Review by kdembler — CHANGES_REQUESTED** [review]
1. The animation for changing animation speed still mentions the old speed in "Speed set to ___" message. For example, I change from 1x to 1.5x and it says "Speed set to 1x"
2. When I click the "CC" button (outside of settings), no animation is applied. Is that desired?

**Comment by rafalpawlow** [comment]
1. This bug doesn't appear in Chrome and Safari, I'll investigate it
2. I'll ask @toiletgranny

**Comment by kdembler** [comment]
1. Oh yeah, that was on Firefox

**Review by kdembler — APPROVED** [review]

**Pull request merged**